src/com/company/Cricket_Game.java

```diff
@@ -0,0 +1,33 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Cricket_Game {
+    public static void main(String[] args)
+    {
+        ArrayList<Team> teams = new ArrayList<Team>();
```

Comment:
new ArrayList<>() will work, since while declaring variable type, you have declared it to be ArrayList of Team